manifests/config_aem.pp

```diff
@@ -2,14 +2,16 @@
   $run_mode,
   $tmp_dir,
   $aem_ssl_port,
+  $host,
```

Comment:
This parameter name is to common and doesn't highlight it's only needed to configure https via Granite. Please rename it to something more specific e.g. `https_hostname`. Also atm it is a required parameter which is not the case if you pass `jetty` as `aem_ssl_method` right ? please assign a default value for it e.g. `undef` or `localhost` Thanks 


We also need to define a default value to keep the backward compatibiliay.